src/core/utils/index.js

```diff
@@ -233,7 +232,7 @@ export function addonHasVersionHistory(addon) {
     throw new Error('addon is required');
   }
 
-  return ![ADDON_TYPE_OPENSEARCH].includes(addon.type);
+  return addon.type;
```

Comment:
I think we should rather return `true`.

Reply:
Thinking about it, I believe we can remove this `addonHasVersionHistory()` function. It is only used in the `AddonMoreInfo` component. I think we can safely remove the call to this function and delete the test case as well.

Reply:
Alright. So, is `addonHasVersionHistory()` used elsewhere other than the test cases. Should we remove it from the constants too?

Reply:
Alright. I will remove the function and its test case.

Reply:
Yes, it is used once in `AddonMoreInfo`. It's fine to remove it entirely and not replace the call in the component (the new behavior of this function should be to return `true` all the time anyway).

Reply:
Alright. Will do that.

Reply:
Removed `addonHasVersionHistory()` itself and from `AddonMoreInfo` component. Consequently the following test cases failed.

 ```
/home/ajin/Desktop/outreachy/addons-frontend/tests/unit/amo/components/TestAddonMoreInfo.js › links to version history if add-on is extension

    toHaveProp must be called on a single node, not an empty node.

      522 |     const history = root.find('.AddonMoreInfo-version-history');
      523 | 
    > 524 |     expect(history).toHaveProp('term', 'Version History');
          |                     ^
      525 |     expect(history.find(Link)).toHaveProp(
      526 |       'to',
      527 |       `/addon/${addon.slug}/versions/`,

      at Object.<anonymous> (tests/unit/amo/components/TestAddonMoreInfo.js:524:21)

  ● /home/ajin/Desktop/outreachy/addons-frontend/tests/unit/amo/components/TestAddonMoreInfo.js › adds a `src` query parameter to the version history link if available in the location

    toHaveProp must be called on a single node, not an empty node.

      543 |     const history = root.find('.AddonMoreInfo-version-history');
      544 | 
    > 545 |     expect(history).toHaveProp('term', 'Version History');
          |                     ^
      546 |     expect(history.find(Link)).toHaveProp(
      547 |       'to',
      548 |       `/addon/${addon.slug}/versions/?src=${src}`,

      at Object.<anonymous> (tests/unit/amo/components/TestAddonMoreInfo.js:545:21)

  ● /home/ajin/Desktop/outreachy/addons-frontend/tests/unit/amo/components/TestAddonMoreInfo.js › links to version history if add-on is a dictionary

    toHaveProp must be called on a single node, not an empty node.

      559 |     const history = root.find('.AddonMoreInfo-version-history');
      560 | 
    > 561 |     expect(history).toHaveProp('term', 'Version History');
          |                     ^
      562 |     expect(history.find(Link)).toHaveProp(
      563 |       'to',
      564 |       `/addon/${addon.slug}/versions/`,

      at Object.<anonymous> (tests/unit/amo/components/TestAddonMoreInfo.js:561:21)

  ● /home/ajin/Desktop/outreachy/addons-frontend/tests/unit/amo/components/TestAddonMoreInfo.js › links to version history if add-on is a language pack

    toHaveProp must be called on a single node, not an empty node.

      575 |     const history = root.find('.AddonMoreInfo-version-history');
      576 | 
    > 577 |     expect(history).toHaveProp('term', 'Version History');
          |                     ^
      578 |     expect(history.find(Link)).toHaveProp(
      579 |       'to',
      580 |       `/addon/${addon.slug}/versions/`,

      at Object.<anonymous> (tests/unit/amo/components/TestAddonMoreInfo.js:577:21)

  ● /home/ajin/Desktop/outreachy/addons-frontend/tests/unit/amo/components/TestAddonMoreInfo.js › links to version history if add-on is a theme

    toHaveProp must be called on a single node, not an empty node.

      588 |     const history = root.find('.AddonMoreInfo-version-history');
      589 | 
    > 590 |     expect(history).toHaveProp('term', 'Version History');
          |                     ^
      591 |     expect(history.find(Link)).toHaveProp(
      592 |       'to',
      593 |       `/addon/${addon.slug}/versions/`,

      at Object.<anonymous> (tests/unit/amo/components/TestAddonMoreInfo.js:590:21)
```

Reply:
okay, we'll investigate these errors later.

Reply:
Sure thing :)

Reply:
I think this is fixed now. :)

Reply:
ok so you can remove the `addonHasVersionHistory` function then :)